CHANGES.rst

```diff
@@ -12,6 +12,22 @@ Yet to be released.
 
 - [Bugfix] #15 #17 - Restore Python 2.6 compatibility. (Thanks to Marco Nenciarini!)
 
+.. attention::
+    Deprecation warnings, these symbols are slated for removal in
+    the next major release.
+
+    - ``caplog.atLevel`` for ``caplog.at_level`` and
+      ``caplog.setLevel`` for ``caplog.setLevel`` are still available
+      but not supported and deprecated since they don't follow the
+      PEP8 convention for method names.
+
+    - ``caplog.text``, ``caplog.records`` and
+      ``caplog.record_tuples`` were turned into properties.
+      They still can be used as regular methods for backwards
+      compatibility.  They will lose the backward compatibility
```

Comment:
Will they? I would rather use less strong statement, like *They still can be used as regular methods for backwards compatibility, but that syntax is considered deprecated and scheduled for removal in the next major release.*